xtask/src/release/bump.rs

```diff
@@ -241,7 +241,22 @@ pub fn bump(bump_version: BumpLevel, dry_run: bool) -> Result<()> {
     let ws = cargo::core::Workspace::new(&Path::new("./Cargo.toml").canonicalize()?, &cargo_context)
         .map_err(|e| eyre!(e.to_string()))?;
 
-    let ws_rust_dep_names_to_bump: Vec<&str> = ws.members().map(|p| p.name().as_str()).collect();
+    let toml_raw_doc_workspace = std::fs::read_to_string(ws.root().join("Cargo.toml"))?;
+    
+    let mut virtual_manifest = toml_raw_doc_workspace.parse::<toml_edit::DocumentMut>()?;
+    let workspace_dependencies_table = virtual_manifest
+        .get_mut("workspace")
+        .unwrap()
+        .get_mut("dependencies")
+        .unwrap()
+        .as_table_like_mut()
+        .unwrap();
```

Comment:
Can you use `expect` instead of unwrap to provide some guidance if it fails for some reason?